test/modules/environment.js

```diff
@@ -66,8 +54,40 @@ module.exports = {
   },
 
   addClientCommands(client) {
-    client.addCommand('loadSettingsPage', function async() {
-      return this.url('file://' + path.join(sourceRootDir, 'src/browser/settings.html')).waitUntilWindowLoaded();
+    client.addCommand('customClick', function async(selector) {
```

Comment:
Why custom `click` command?
Finding it to be much more reliable where WebdriverIO `click` is not.